data-serving/scripts/converters.py

```diff
@@ -182,6 +182,33 @@ def convert_location(id: str, location_id: float, country: str, adminL1: str,
     return location
 
 
+def convert_dictionary_field(
+        id: str, field_name: str, value: str) -> Dict[
+        str, List[str]]:
+    if pd.isna(value):
+        return None
+    if type(value) is not str:
+        logging.warning(
+            '[%s] [field_name] invalid value %s', id, value)
+        return None
+
+    is_comma_delimited = value.find(',') >= 0
+    is_colon_delimited = value.find(':') >= 0
+
+    if is_comma_delimited:
+        if is_colon_delimited:
+            logging.warning(
+                '[%s] [field_name] two delimiters detected, using comma %s', id,
+                value)
+
+        return {'provided': trim_string_array(value.split(','))}
+    if is_colon_delimited:
```

Comment:
I can't recall -- any data delimited by something besides these?

Reply:
not that I've seen yet, but many of these fields are filled out pretty infrequently so I expect we can do a manual inspection phase once the bulk of the converting logic is, and touch things up then